WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1284,16 +1284,14 @@ - (UIAlertAction *)quickSaveAlertAction
             return [UIAlertAction actionWithTitle:NSLocalizedString(@"Publish Now", "Title of button allowing the user to immediately publish the post they are editing.")
                                             style:UIAlertActionStyleDestructive
                                           handler:^(UIAlertAction * _Nonnull action) {
-                                              self.post.status = PostStatusPublish;
```

Comment:
Hmm, I think we have to keep these status changes since `publishImmediately` only changes the `dateCreated` and `dateModified`.